docs/upgrade.md

```diff
@@ -99,6 +99,13 @@ experimental_features:
   groups_enabled: false
 ```
 
+## Change in behaviour for PostgreSQL databases with unsafe locale
+
+Synapse now refuses to start when using PostgreSQL with incorrect values of `COLLATE` and
```

Comment:
```suggestion
Synapse now refuses to start when using PostgreSQL with non-`C` values for `COLLATE` and
```
Maybe this is clearer? Thanks for doing this btw, I didn't think to do so!

Reply:
No worries :)